app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -352,7 +357,7 @@ class ThreadFragment : Fragment() {
     }
 
     private companion object {
-        const val COLLAPSE_TITLE_THRESHOLD = 0.5
+        const val COLLAPSE_TITLE_THRESHOLD = 0.666
```

Comment:
Why is this threshold change better or needed ?

Reply:
With `0.5`, there is some frames where the subject is totally hidden.
It feels more smooth this way.
![image](https://user-images.githubusercontent.com/1788629/233351493-cb402760-b849-4f0d-8083-7530983fa6f0.png)